src/main/resources/hudson/plugins/git/GitChangeSetList/index.jelly

```diff
@@ -25,7 +25,7 @@
               <j:if test="${cslink==null}">
                 ${cs.id}
               </j:if>
-              by <a href="${rootURL}/${cs.author.url}/">${cs.author}</a>
+              by <a href="${rootURL}/${cs.author.url}/">${cs.author.displayName}</a>
```

Comment:
Could also use `fullName` if that would be clearer in the code
```suggestion
              by <a href="${rootURL}/${cs.author.url}/">${cs.author.fullName}</a>
```

Reply:
I don't think this makes sense unless you specifically want to show the full name if we ever change what the display name is.

Reply:
I don't understand your reference to model-links @daniel-beck .  Can you point me to an example or other guidance on how this should be done better?

Reply:
Links to `ModelObject`s are typically `class="model-link inside"`, then they get the little triangle with the dropdown menu.

Reply:
Thanks for the pointer.  I tested with `class="model-link inside"` in a quick mock-up and will remember the technique for other areas.

I think the changes list in this context is not a likely place where the user will benefit from a drop down that gives them access to builds, views, credentials, and user configuration.

## Looked like this (without expanding the dropdown menu):

![Screenshot 2020-09-24 081431-a](https://user-images.githubusercontent.com/156685/94157599-a81b2480-fe3e-11ea-91ec-398aa4192e93.png)

## Looked like this (with expanded dropdown menu):

![Screenshot 2020-09-24 081524](https://user-images.githubusercontent.com/156685/94157646-b6694080-fe3e-11ea-9703-be8ed6379f21.png)